### DISCUSSION TITLE
On/Off control stops working

### DISCUSSION THREAD

**Discussion by jacktose** [darkreader/darkreader#6959] [Q&A] [answered]
Lately I can't reliably turn the extension on after I turn it off.

**1.** I start like so, everything working:
![dr1](https://user-images.githubusercontent.com/12414794/135342759-e6165201-01e7-49f7-a05c-f453ab017f0d.png)

**2.** I click Off and the control panel and page change as expected:
![dr2](https://user-images.githubusercontent.com/12414794/135342762-05016508-a2df-4d63-a2c9-d86feb4c5b47.png)

**3.** I close the control panel and reopen it (without clicking the On/Off control), and it shows On and “extension is enabled,” but the page is still light:
![dr3](https://user-images.githubusercontent.com/12414794/135342764-688fb529-e102-425a-ad9f-a5d5579592f0.png)

**4.** After this, I can click On and Off any number of times, but it will only toggle between states 2 and 3. It never actually darkens the page again.

**5.** I disable/enable the extension itself (or restart the browser, I think), and that takes me back to state 1.

Any help or suggestions are welcome!

Dark Reader 4.9.37.1; Firefox 92.0.1; Windows 10

**Comment by Gusted** [comment, marked as answer]
#6873
